vizro-core/src/vizro/_themes/_templates/template_dark.py

```diff
@@ -71,31 +71,7 @@ def create_template_dark() -> Template:
     }
 
     # CHART TYPES
-    template_dark.data.bar = [
-        go.Bar(
-            marker={"line": {"color": template_dark["layout"]["paper_bgcolor"], "width": 1}},
-            textfont={"color": COLORS["WHITE_55"]},
-        )
-    ]
-
-    template_dark.data.table = [
```

Comment:
Removed the go.Table as this was always just intended as a temporary solution until we had the dash_data_table and/or dash_ag_grid in place